app/main.js

```diff
@@ -87,6 +87,10 @@ ipcMain.on('logGetEntries', (event, arg) => {
   event.returnValue = proxy.getLogEntries();
 })
 
+ipcMain.on('clearLogEntries', (event, arg) => {
+  event.returnValue = proxy.clearLogs();
```

Comment:
You dont need to return anything here. You should emit the logUpdated event with `win.webContents.send('logupdated', entry)`. Maybe its better if we send the complete proxy.logEntries instead of only the last created one. You would need to change that in the Logs.js too, because right now that expects only a single entry and not the entire array.